packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -151,16 +150,6 @@ class CupertinoPageRoute<T> extends PageRoute<T> {
   @override
   final bool maintainState;
 
-  /// The route that owns this one.
-  ///
-  /// The [MaterialPageRoute] creates a [CupertinoPageRoute] to handle iOS-style
-  /// navigation. When this happens, the [MaterialPageRoute] is the [hostRoute]
-  /// of this [CupertinoPageRoute].
-  ///
-  /// The [hostRoute] is responsible for calling [dispose] on the route. When
-  /// there is a [hostRoute], the [CupertinoPageRoute] must not be [install]ed.
-  final PageRoute<T> hostRoute;
-
```

Comment:
This is technically a breaking change so we should definitely mention it on flutter-dev, but I find it hard to imagine that it's going to break any actual apps since it doesn't really _do_ anything.

Reply:
I sent a warning about this change to flutter-dev, subject "Proposed changes to the CupertinoPageRoute class", on 9/10. Will definitely send a definitive message when this PR lands.

Reply:
Great, thanks.